driver-core/src/main/com/mongodb/connection/QueryResult.java

```diff
@@ -30,6 +30,8 @@
  */
 @Deprecated
 public class QueryResult<T> {
+    private static final String POST_BATCH_RESUME_TOKEN = "postBatchResumeToken";
```

Comment:
This field can now be reverted to as it is on master

Reply:
Yeap: -field +file

Reply:
The field has been removed.